project-keeper/src/main/java/com/exasol/projectkeeper/validators/pom/PomFileValidator.java

```diff
@@ -57,12 +57,28 @@ public PomFileValidator(final Path projectDirectory, final Collection<ProjectKee
             if (this.enabledModules.contains(ProjectKeeperModule.JAR_ARTIFACT)) {
                 findings.addAll(validateAssemblyPlugin(pom, this.projectDirectory.relativize(this.pomFilePath)));
             }
-            return findings;
+            return wrapFindingsWithGroupThatWritesPom(pom, findings);
         } catch (final InvalidPomException exception) {
             return List.of(SimpleValidationFinding.withMessage(exception.getMessage()).build());
         }
     }
 
+    private List<ValidationFinding> wrapFindingsWithGroupThatWritesPom(final Document pom,
+            final List<ValidationFinding> findings) {
+        if (hasFindingWithFix(findings)) {
+            return List.of(
+                    new ValidationFindingGroup(findings, () -> new PomFileIO().writePomFile(pom, this.pomFilePath)));
+        } else {
+            return findings;
+        }
+    }
+
+    private boolean hasFindingWithFix(final List<ValidationFinding> findings) {
+        return findings.stream().anyMatch(finding -> //
```

Comment:
`//` comment left over?

Reply:
We do that for manual formatting. By adding `//` the formatter can't remove the linebreak...